wip/openapi-maven-plugin-0.0.28.buildspec

```diff
@@ -0,0 +1,32 @@
+# see instructions https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/doc/BUILDSPEC.md
+
+# Central Repository coordinates for the Reference release (for multi-module, pick an arbitrary module)
+groupId=io.github.kbuntrock
+artifactId=openapi-maven-plugin
+version=0.0.28
+
+display=${groupId}:${artifactId}
+
+# Source code
+gitRepo=https://github.com/kbuntrock/${artifactId}.git
+gitTag=${version}
+
+# Rebuild environment prerequisites
+tool=mvn-3.9.9
+jdk=17
+newline=lf
+
+# optional:
+#umask=022
+#timezone="Etc/GMT"
+#locale="en_US"
+
+# Rebuild command
+command="mvn clean package -Pprerelease -DskipTests -f openapi-maven-plugin"
+
+# Location of the buildinfo file generated during rebuild (by artifact:compare for Maven) to record output fingerprints
+buildinfo=target/${artifactId}-${version}.buildinfo
```

Comment:
given you have `-f openapi-maven-plugin` in the mvn command, you need

    buildinfo=openapi-maven-plugin/target/${artifactId}-${version}.buildinfo